### PR TITLE
Bake BE4 plugin task

### PR DESCRIPTION
**Pull request by stefanorosanelli**
This PR adds a new task to `bake` a new BEdita plugin via

```bash
bin/cake bake bedita_plugin MyPlugin
```

This will create an empty plugin structure ready for use as BEdita plugin with some simple customizations of regular cake plugins.

Plugin is not automatically loaded in bootstrap like classic plugins, but needs to be included in `Plugins` config.

### CODE REVIEW

File: src/Shell/Task/BeditaPluginTask.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * The BEdita Plugin Task handles creating an empty BEdita4 plugin, ready to be used
+ */
+class BeditaPluginTask extends BakeTask
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected function _modifyBootstrap($plugin, $hasAutoloader)
+    {
```
Comment (fquffio) [inline src/Shell/Task/BeditaPluginTask.php:15]: I'd add a line of comment to explain why this method is being overwritten empty. Something like `// Avoid updating bootstrap.php` would be just fine IMHO.

**Review by fquffio — APPROVED** [review]
Ok for me, but wouldn't this template fit better in the DevTools plugin? This is something that's meant for development purposes solely, right?

**Comment by stefanorosanelli** [comment]
Yes, we could move to `DevTools`.
I'll create a new PR in that repo, then I will remove this one

**Comment by stefanorosanelli** [comment]
Close as obsolete
Moved to bedita/dev-tools#4

**Pull request closed without merging**